### PR TITLE
[DOCS-9559] Add partition selection section to aws manual setup docs

### PR DESCRIPTION
**Pull request by saleelsaptarshi-dd**
### What does this PR do? What is the motivation?

A change to the AWS account setup flow has been released. This PR updates AWS manual account setup docs with instructions for selecting an AWS partition. 
Also removes outdated text suggesting that access keys are only supported for gov and china regions. 

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/integrations/guide/aws-manual-setup.md [hunk 1]
```diff
@@ -100,12 +106,13 @@ Ensure to leave `Require MFA` disabled. For more details, see the [How to use an
 
 ### Complete the setup in Datadog
 
-18. Return to the AWS integration configuration page for manually adding an account in Datadog that you had open in another tab. Click the checkbox to confirm the Datadog IAM role was added to the AWS account.
+18. Return to the AWS integration configuration page for manually adding an account in Datadog that you had open in another tab. Click the checkbox to confirm the Datadog IAM role was added to the AWS account. 20
```
Comment (ktmq) [inline content/en/integrations/guide/aws-manual-setup.md:109]:
```suggestion
18. Return to the AWS integration configuration page for manually adding an account in Datadog that you had open in another tab. Click the checkbox to confirm the Datadog IAM role was added to the AWS account.
```

**Comment by buraizu** [comment]
Created DOCS-9559 to review

File: content/en/integrations/guide/aws-manual-setup.md [hunk 2]
```diff
@@ -62,14 +62,20 @@ To set up the AWS integration manually, create an IAM policy and IAM role in you
...
+{{< /site-region >}}
+3. Copy the `AWS External ID`. For more information about the external ID, read the [IAM User Guide][2].
   **Note**: The External ID remains available and is not regenerated for 48 hours, unless explicitly changed by a user or another AWS account is added to Datadog during this period. You can return to the **Add New AWS Account** page within that time period to complete the process of adding an account without the External ID changing.
 
 ### AWS IAM policy for Datadog
 Create an IAM policy for the Datadog role in your AWS account with the [necessary permissions](#aws-integration-iam-policy) to take advantage of every AWS integration offered by Datadog. As other components are added to an integration, these permissions may change.
 
 3. Create a new policy in the AWS [IAM Console][3].
-4. Select the **JSON** tab. Paste the [permission policies](#aws-integration-iam-policy) in the textbox.  
+4. Select the **JSON** tab. Paste the [permission policies](#aws-integration-iam-policy) in the textbox.
```
Comment (buraizu) [inline content/en/integrations/guide/aws-manual-setup.md:78]:
```suggestion
4. Select the **JSON** tab. Paste the [permission policies](#aws-integration-iam-policy) in the textbox.<br>
```

File: content/en/integrations/guide/aws-manual-setup.md [hunk 3]
```diff
@@ -102,10 +108,11 @@ Ensure to leave `Require MFA` disabled. For more details, see the [How to use an
 
 18. Return to the AWS integration configuration page for manually adding an account in Datadog that you had open in another tab. Click the checkbox to confirm the Datadog IAM role was added to the AWS account.
 19. Enter the account ID **without dashes**, for example: `123456789012`. Your Account ID can be found in the ARN of the role created for Datadog.
-20. Enter the name of the role created in the previous section, and click **Save**.  
+20. Choose which AWS partition your AWS account is scoped to. The partition is either `aws` for commercial regions, `aws-cn` for China, or `aws-us-gov` for GovCloud. Refer to [AWS documentation][9] for more information on AWS partitions.
```
Comment (buraizu) [inline content/en/integrations/guide/aws-manual-setup.md:111]:
```suggestion
20. Choose which AWS partition your AWS account is scoped to. The partition is either `aws` for commercial regions, `aws-cn` for China*, or `aws-us-gov` for GovCloud. See [Partitions][9] in the AWS documentation for more information.
```

File: content/en/integrations/guide/aws-manual-setup.md [hunk 4]
```diff
@@ -102,10 +108,11 @@ Ensure to leave `Require MFA` disabled. For more details, see the [How to use an
...
+21. Enter the name of the role created in the previous section, and click **Save**.
   **Note**: The role name you enter in the integration tile is case sensitive and must exactly match the role name in AWS.
-21. If there is a [Datadog is not authorized to perform sts:AssumeRole][6] error, follow the troubleshooting steps recommended in the UI, or read the [troubleshooting guide][6].
-22. Wait up to 10 minutes for data to start being collected, and then view the out-of-the-box <a href="https://app.datadoghq.com/screen/integration/7/aws-overview" target="_blank">AWS Overview Dashboard</a> to see metrics sent by your AWS services and infrastructure.
+22. If there is a [Datadog is not authorized to perform sts:AssumeRole][6] error, follow the troubleshooting steps recommended in the UI, or read the [troubleshooting guide][6].
+23. Wait up to 10 minutes for data to start being collected, and then view the out-of-the-box <a href="https://app.datadoghq.com/screen/integration/7/aws-overview" target="_blank">AWS Overview Dashboard</a> to see metrics sent by your AWS services and infrastructure.
```
Comment (buraizu) [inline content/en/integrations/guide/aws-manual-setup.md:115]:
```suggestion
23. Wait up to 10 minutes for data to start being collected, and then view the out-of-the-box <a href="https://app.datadoghq.com/screen/integration/7/aws-overview" target="_blank">AWS Overview Dashboard</a> to see metrics sent by your AWS services and infrastructure.

*\* All use of Datadog Services in (or in connection with environments within) mainland China is subject to the disclaimer published in the [Restricted Service Locations][62] section on our website.*
```

File: content/en/integrations/guide/aws-manual-setup.md (hunk 2 above)
Comment (buraizu) [reply content/en/integrations/guide/aws-manual-setup.md:78]: The double spaces at the end of the line here create a line break in markdown, so the note renders indented on the next line. If you want to delete these spaces, they should be replaced with a `<br>` element to achieve the same effect.

File: content/en/integrations/guide/aws-manual-setup.md (hunk 3 above)
Comment (buraizu) [reply content/en/integrations/guide/aws-manual-setup.md:111]: Per the [style guide on link text](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#links)

File: content/en/integrations/guide/aws-manual-setup.md (hunk 4 above)
Comment (buraizu) [reply content/en/integrations/guide/aws-manual-setup.md:115]: In addition to adding this disclaimer here, it should be deleted from line `146`

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR, requesting a few minor edits

**Review by buraizu — APPROVED** [review]

**Pull request merged**